src/KingSystem/System/QuestMgr.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <container/seadPtrArray.h>
+#include <heap/seadDisposer.h>
+#include <prim/seadSafeString.h>
+#include "KingSystem/System/Quest.h"
+#include "agl/Utils/aglParameter.h"
+
+namespace ksys {
+
+class QuestMgr {
```

Comment:
According to symbols it looks like this was called qst::Manager ("Manager" in the ksys::qst namespace). For consistency with qst::Quest, could you rename this class please?